test/createComponent.spec.js

```diff
@@ -26,8 +36,14 @@ describe('createComponent', function () {
         componentDidMount: sinon.match.func,
         componentWillUnmount: sinon.match.func
       });
-    expect(myComponentInstance).to.be.instanceof(MyComponent);
+  });
+
+  it('is a valid React component and a MyComponent\'s instance', function () {
     expect('isReactComponent' in Object.getPrototypeOf(myComponentInstance)).to.be.equal(true);
+    expect(myComponentInstance).to.be.instanceof(MyComponent);
+  });
+
+  it('has the spec\'s functions', function () {
```

Comment:
Linting rule: Strings should be either single quoted or, in case we need to concatenate with variables, we should use template literals.

But I can improve it to be 'has the functions of the spec'